### PR TITLE
Switch to version of dart-style that does not pollute global

### PR DESCRIPTION
**Pull request by jacob314**
Let me know where you would like the regression test.

### CODE REVIEW

File: test/main_test.ts [hunk 1]
```diff
@@ -83,3 +83,11 @@ class A {
     });
   });
 });
+
+describe('regression', () => {
```
Comment (mprobst) [inline test/main_test.ts]: I think it's fine to leave out the test. It's a corner case internal bug in a dependency - this should be a test in dart-style, but not in ts2dart.

Comment (jacob314) [inline test/main_test.ts]: Reverted the test file. This is really a test for the library that wraps dart2js output so it can run on NodeJS.

**Comment by vikerman** [comment]
LGTM. Will merge it soon. Thanks

**Comment by mprobst** [comment]
Merged in c313fbb.
